lib/notify.js

```diff
@@ -49,12 +50,18 @@ module.exports = CoreObject.extend({
         .catch(function(error) {
           var errorMessage = serviceKey + ' => ' + error;
 
+          if (critical === true) {
+            return Promise.reject(error);
+          }
           plugin.log(errorMessage, { color: 'red' });
         });
     } else {
       var warningMessage = 'No request issued! Request options invalid! You have to specify `url`, `headers`, `method` and `body`.';
-      plugin.log(serviceKey+' => '+warningMessage, { color: 'yellow', verbose: true });
 
+      if (critical === true) {
```

Comment:
sorry i got confused. @LevelbossMike mentioned `critical === true` in the last comment and i thought he preferrd like that